docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1488,6 +1488,8 @@ Note: If you use PostgreSQL database as IBM FHIR Server data store or the JavaBa
 
 For more information about Liberty JavaBatch configuration, please refer to [IBM WebSphere Liberty Java Batch White paper](https://www-03.ibm.com/support/techdocs/atsmastr.nsf/webindex/wp102544).
 
+If you are running in a Kubernetes deployment, be sure to set the environment variable MY_POD_NAME to `metadata.name` as shown in [deployment.yaml](https://github.com/Alvearie/alvearie-helm/blob/main/charts/ibm-fhir-server/templates/deployment.yaml#L159). This setting allows the stopping and deleting of jobs on all the hosts in the deployment.
```

Comment:
wish there was a better way to keep these links up-to-date...but probably easiest for now just to make it right for this point-in-time
```suggestion
If you are running in a Kubernetes deployment, be sure to set the environment variable MY_POD_NAME to `metadata.name` as shown in [deployment.yaml](https://github.com/Alvearie/alvearie-helm/blob/main/charts/ibm-fhir-server/templates/deployment.yaml#L176). This setting allows the stopping and deleting of jobs on all the hosts in the deployment.
```